tests/stubtest_allowlists/py310.txt

```diff
@@ -183,3 +194,17 @@ typing.NewType.__call__
 typing_extensions\.Final
 typing\.NamedTuple
 typing\.Annotated
+
+# We lie about the existence of these methods
+.*.__buffer__
+.*.__release_buffer__
+
+# Removed in 3.12
+distutils\..*
```

Comment:
I would have preferred to not do this. I feel like whenever we've used wildcard regex allowlist entries in the past, it's ended up causing us pain and confusion further down the line (https://github.com/python/typeshed/pull/9900; https://github.com/python/typeshed/pull/10205; there have been others as well).

If we _have_ to do this, then we may as well revert the changes that were made to `stubtest_stdlib.py` in #9734. The `venv` module is really slow on Windows, so those changes basically doubled the time it takes for me to run the test locally.

Reply:
I agree in general, but I have basically no further interest in the accuracy of the distutil stubs, especially since most users aren't even going to get distutils when they `import distutils`. So that points to undo #9734. But I definitely wouldn't say "we _have_ to do this"... let me know which one you want and I'll make the PR.

Reply:
Yeah, that all makes sense :-) let's go for undoing #9735, then.

Reply:
Longer term, I guess we might want to look at getting type checkers to somehow recognise that if a user does `import distutils` with `setuptools` installed, they'll actually be getting `setuptools._distutils`... but the idea of adding that kind of special-casing to type checkers *really* doesn't fill me with joy.....